source/ExpandablePicker.js

```diff
@@ -370,6 +370,9 @@
 			this.inherited(arguments);
 			this.$.currentValue.setShowing(!this.open);
 			this.setActive(this.getOpen());
+			if(this.open) {
+				this.$.drawer.render();	
```

Comment:
I could not use reflow on the parent, since it doesn't notify the children. I did create this PR to add a reflow call down through children https://github.com/enyojs/enyo/pull/1212